packages/manager/modules/cloud-connect/src/style.scss

```diff
@@ -0,0 +1,5 @@
+cloud-connect {
```

Comment:
why applying box shadow none on a oui-tile ? i would prefer to let the default oui-tile style for consistency (even if the mockup does not display a drop shadow)

Reply:
It looks a bit odd as i am adding `tile` inside a `tile`. Ill remove for now and see if it goes through QA